src/server/api/campaign.js

```diff
@@ -486,6 +486,10 @@ export const resolvers = {
               "campaign_contact.assignment_id",
               "assignment.id"
             )
+            .where(
```

Comment:
Since a user can be part of multiple organizations, this WHERE filter is required for correct contact counts calculation of a campaign